eq-author/src/App/QuestionnairesPage/QuestionnairesView/index.test.js

```diff
@@ -373,5 +373,64 @@ describe("QuestionnairesView", () => {
         expect(getByText("2 of 2")).toBeTruthy();
       });
     });
+    describe("Sorting", () => {
+      let props;
+      beforeEach(() => {
+        const questionnaires = [
+          buildQuestionnaire(4),
+          buildQuestionnaire(2),
+          buildQuestionnaire(1),
+          buildQuestionnaire(3),
+          buildQuestionnaire(5),
+        ];
+        const user = {
+          id: "3",
+          name: "Foo",
+          email: "foo@bar.com",
+          displayName: "Foo",
+        };
+
+        props = {
+          questionnaires,
+          user,
+          onDeleteQuestionnaire: jest.fn(),
+          onDuplicateQuestionnaire: jest.fn(),
+          onCreateQuestionnaire: jest.fn(),
+        };
+      });
+      afterEach(() => {
```

Comment:
This should be happening already as this is within the `QuestionnairesView` describe.

Reply:
Removed this as it was happening already.